Packs/GraphQL/Integrations/GraphQL/GraphQL.py

```diff
@@ -0,0 +1,56 @@
+from typing import Dict
+
+import urllib3
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+def execute_query(client: Client, args: Dict) -> CommandResults:
+    query = gql(args.get('query'))
```

Comment:
lets add an argument, `key_field`, if provided then you will pass it to `outputs_key_field`